prov/gni/include/gnix_av.h

```diff
@@ -37,19 +37,35 @@
 #include "gnix.h"
 
 /*
+ * this structure should ideally be as compact
+ * as possible, since its looked up in the critical
+ * path for FI_EP_RDM EPs that use FI_AV_MAP
+ */
+
+struct gnix_av_addr_entry {
+	struct gnix_address gnix_addr;
+	struct {
+		uint32_t name_type : 8;
+		uint32_t cm_nic_cdm_id : 24;
+		uint32_t cookie;
+	};
+};
```

Comment:
So just an gnix_ep_name without the extra padding?  Why not just remove the padding from the ep name?